assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -35,23 +35,38 @@ import { BUTTON_TYPES, primaryPaths, secondaryPaths } from '../../constants';
 import {
   AppInfo,
   Content,
-  LeftRailContainer,
   LogoPlaceholder,
   NavLink,
   Rule,
 } from './navigationComponents';
 
 export const AppFrame = styled.div`
-  display: flex;
-  flex-direction: row;
-  height: inherit;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
 `;
 
 export const PageContent = styled.div`
-  position: relative;
-  width: 100%;
-  padding-left: max(15%, 190px);
-  height: inherit;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: max(15%, 190px);
+`;
+
+export const LeftRailContainer = styled.nav`
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  top: 0;
+  bottom: 0;
+  width: max(15%, 190px);
+  background: ${({ theme }) => theme.colors.white};
+  border-right: ${({ theme }) => theme.leftRail.border};
+  z-index: 2;
```

Comment:
do we want z-index in the theme so that this 2 here and in the nav can be seen as the same?

Reply:
We do have a `Z_INDEX` constant to store our site wide z-indices.